src/System.Drawing.Common/src/Unix/System.Drawing.Imaging/Metafile.cs

```diff
@@ -68,7 +68,7 @@ internal Metafile(IntPtr ptr, Stream stream)
         public Metafile(Stream stream)
         {
             if (stream == null)
-                throw new ArgumentException("stream");
+                throw new ArgumentNullException("stream");
```

Comment:
Nit: at some point it'd be good to go through and use `nameof` for args.  That was previously done for most exceptions throughout the corefx codebase, but code ported to corefx since missed out on that.  That's not needed for this PR, though.